sentry-ruby/lib/sentry/backtrace.rb

```diff
@@ -35,25 +38,33 @@ class Line
       # Parses a single line of a given backtrace
       # @param [String] unparsed_line The raw line from +caller+ or some backtrace
       # @return [Line] The parsed backtrace line
-      def self.parse(unparsed_line, in_app_pattern = nil)
-        ruby_match = unparsed_line.match(RUBY_INPUT_FORMAT)
-        if ruby_match
-          _, file, number, _, method = ruby_match.to_a
-          file.sub!(/\.class$/, RB_EXTENSION)
-          module_name = nil
-        else
+      def self.parse(unparsed_line, in_app_pattern = nil, cleaned_line = nil)
+        if RUBY_PLATFORM == "java"
```

Comment:
we only need to run this branch if we are in java land so we're saving some useless cycles now